spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -384,6 +394,13 @@
       expect(subject).to receive(:ab_finished).with(:demo)
       get :new
     end
+
+    it 'tracks the pageview' do
```

Comment:
I probably won't be able to do this before I go on vacation. If y'all want to play with metrics while I'm gone, feel free to make this change. Or, we can merge this as is, and then I'll clean it up when I get back.